presto-main/src/main/java/com/facebook/presto/cost/HistoryBasedPlanStatisticsCalculator.java

```diff
@@ -191,12 +188,12 @@ private PlanNodeStatsEstimate getStatistics(PlanNode planNode, Session session,
         catch (ExecutionException e) {
             throw new RuntimeException(format("Unable to get plan statistics for %s", planNode), e.getCause());
         }
-        double historyMatchingThreshold = getHistoryInputTableStatisticsMatchingThreshold(session) > 0 ? getHistoryInputTableStatisticsMatchingThreshold(session) : config.getHistoryMatchingThreshold();
+        double historyMatchingThreshold = getHistoryInputTableStatisticsMatchingThreshold(session);
```

Comment:
I guess this is from your old change

Reply:
Yes, it belongs to the previous PR. Only the last one of all three commits here belongs to this PR.